problems/fizzbuzz/cpp/matthewrolls/script.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+echo Enter number
```

Comment:
do not ask the number here during the script.  this should be passed as an argument to the shell script:  `./script.sh 16`